app/views/partials/card/returns/potential_returns.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.AuthenticatedRequest
+@import models.CtEnrolment
+@import config.FrontendAppConfig
+
+@(appConfig: FrontendAppConfig, ctEnrolment: CtEnrolment)(implicit request: AuthenticatedRequest[_], messages: Messages)
+
```

Comment:
Why are you passing the ctEnrolment when it isn't used?

Reply:
Removed it. Thank you.